.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = C901, E203, E266, E501, E731, W503
+ignore = C901, E203, E231, E266, E501, E731, W503
```

Comment:
Black/flake8 disagreement about spaces after trailing comma. Black wins over Flake8, same as in the dash-table.